docs/src/accelerators.md

```diff
@@ -0,0 +1,100 @@
+# Accelerators
+
+Here, we introduce the Accelerator structs in `Accelerators.jl`, which have been adapted from gradient-based methods in order to accelerate gradient-free ensemble Kalman processes.
+
+While developed for use with ensemble Kalman inversion (EKI), Accelerators have been shown to accelerate convergence for ensemble transform Kalman inversion (ETKI) as well. They are also implemented for use with unscented Kalman inversion (UKI) and the ensemble Kalman sampler (EKS); however, acceleration is experimental for these methods, and often results in instability.
+
+## "Momentum" Acceleration in Gradient Descent
+
+In traditional gradient descent, one iteratively solves for $x^*$, the minimizer of a function $f(x)$, by performing the update step 
+
+```math
+x_{k+1} = x_{k} + \alpha  \nabla f(x_{k}), 
+```
+
+where $\alpha$ is a step size parameter.
+In 1983, Nesterov's momentum method was introduced to accelerate gradient descent. In the modified algorithm, the update step becomes 
```

Comment:
Could we instead say "Nesterov acceleration (a.k.a. Nesterov Momentum)", and then only refer to acceleration rather than momentum. I believe the later field avoid using the term momentum any more.